gazebo_plugins/src/gazebo_ros_camera_utils.cpp

```diff
@@ -175,13 +175,34 @@ void GazeboRosCameraUtils::Load(sensors::SensorPtr _parent,
   else
     this->cy_ = this->sdf->Get<double>("Cy");
 
-  if (!this->sdf->HasElement("focalLength"))
+  if (this->sdf->HasElement("focalLength"))
   {
-    ROS_DEBUG("Camera plugin missing <focalLength>, defaults to 0");
-    this->focal_length_= 0;
+    this->focal_length_x_ = this->sdf->Get<double>("focalLength");
+    this->focal_length_y_ = this->focal_length_x_;
+  }
+
+  if (!this->sdf->HasElement("focalLength_x"))
```

Comment:
Corrected sdf parameters names.
In relation to the warnings, it only outputs msgs if the focalLength and focalLengthX / focalLengthY aren't specified.

Thanks for the feedback :)